Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitClient.swift

```diff
@@ -29,13 +33,16 @@ import BraintreeCore
         self.webAuthenticationSession = webAuthenticationSession
         self.sepaDirectDebitAPI = sepaDirectDebitAPI
     }
+
+    // MARK: - Public Methods
     
     /// Initiates an `ASWebAuthenticationSession` to display a mandate to the user. Upon successful mandate creation, tokenizes the payment method and returns a result
     /// - Parameters:
-    ///   - request: a BTSEPADebitRequest
+    ///   - request: a `BTSEPADebitRequest`
     ///   - completion: This completion will be invoked exactly once when tokenization is complete or an error occurs
+    @objc(tokenizeWithSEPADirectDebitRequest:completion:)
     public func tokenize(
-        request: BTSEPADirectDebitRequest,
+        _ request: BTSEPADirectDebitRequest,
```

Comment:
Does this need to go in Changelog?

Reply:
Yes- it is in there already. In the files changed for this PR.

Reply:
Oop, I c it now